lib/glific/tickets.ex

```diff
@@ -298,4 +298,21 @@ defmodule Glific.Tickets do
 
     true
   end
+
+  @doc """
+  Closing tickets in bulk on the basis of topic
+  """
+  @spec bulk_closure_ticket(map()) :: boolean
```

Comment:
Just update the naming conventions overall. The APIs are not for closure of tickets specifically. They are for updating the status based on the topic.